src/components/calcite-color-picker/calcite-color-picker.scss

```diff
@@ -84,95 +74,94 @@ $gap--large: 12px;
 
 :host([appearance="minimal"]) {
   .container {
-    border: none;
+    @apply border-none;
   }
 }
 
 .container {
-  display: inline-block;
-  border: 1px solid var(--calcite-ui-border-1);
-  background-color: var(--calcite-ui-foreground-1);
+  @apply inline-block
+  border
+  border-solid
+  border-color-1
+  bg-foreground-1;
 }
 
 .color-field-and-slider {
-  margin-bottom: -16px;
+  @apply -mb-4;
 
   &--interactive {
-    cursor: pointer;
+    @apply cursor-pointer;
   }
 }
 
 .control-section {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  @apply flex flex-wrap;
 }
 
 .section {
-  padding: 0 $section-padding $section-padding;
+  @apply pt-0 px-3 pb-3;
 
   &:first-of-type {
-    padding-top: $section-padding;
+    @apply pt-3;
   }
 }
 
 .color-hex-options,
 .section--split {
-  flex-grow: 1;
+  @apply flex-grow;
 }
 
 .header {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  color: var(--calcite-ui-text-3);
-  font-weight: 500;
-  @include font-size(-2);
+  @apply flex
+    items-center
+    justify-between
+    text-color-3
+    font-medium
+    text--2-wrap;
 
   &.header--underlined {
-    border-bottom: 1px solid var(--calcite-ui-border-1);
+    @apply border-0
+      border-b
+      border-solid
+      border-color-1;
   }
 }
 
 .header--hex {
-  line-height: 1.5;
-  @include font-size(-2);
-  padding: $section-padding 0 15px;
+  @apply leading-normal
+    text--2h
+    pt-3 px-0 pb-1;
 }
 
 .control {
-  margin-top: 8px;
+  @apply pt-2;
 }
 
 .channels {
-  display: flex;
-  justify-content: space-between;
+  @apply flex justify-between;
 }
 
 .channel {
   width: 31%;
```

Comment:
:eyes:

Reply:
changing it to a tailwind value breaks the design. We can look at this later.